webapp-django/crashstats/signature/views.py

```diff
@@ -296,3 +301,66 @@ def signature_comments(request):
     data['query'] = search_results
 
     return render(request, 'signature/signature_comments.html', data)
+
+
+@waffle_switch('signature-report')
+def signature_graph(request):
+    '''Return a graph of crashes/ADU against build date. '''
+
+    return render(request, 'signature/signature_graph.html')
```

Comment:
Remove this and instead include it in the ``signature_report`` template, that will save an unnecessary HTTP request.